### DISCUSSION TITLE
Issie with new VH_plus world

### DISCUSSION THREAD

**Discussion by JaworB** [lloesche/valheim-server-docker#616] [Q&A]
Hi there,
I have an issue with running server with VH_plus enabled. I've used docker compose and variables files from my previous world (last run about two months ago). But I was unable to start it once again.
After that I've moved config, and data to other location to start from scratch. Unfortunately every startup stops here

> 2023-04-05 20:17:15,815 INFO success: valheim-server entered RUNNING state, process has stayed up for > than 10 seconds (startsecs)
> Apr  5 20:17:20 supervisord: valheim-server /usr/local/bin/valheim-server: line 123:  1306 Segmentation fault      (core dumped) LD_PRELOAD=$SERVER_LD_PRELOAD "$valheim_server" -nographics -batchmode -name "$SERVER_NAME" -port "$SERVER_PORT" -world "$WORLD_NAME" -public "$SERVER_PUBLIC" "${password_args[@]}" $SERVER_ARGS > >(filter) 2>&1
> Apr  5 20:17:20 supervisord: valheim-server DEBUG - [1301] - Valheim server with PID 1306 stopped
> 2023-04-05 20:17:20,983 INFO reaped unknown pid 1308 (exit status 0)
> Apr  5 20:17:20 supervisord: valheim-server INFO - Releasing PID file /var/run/valheim/valheim-server.pid
> Apr  5 20:17:20 supervisord: valheim-server INFO - Shutdown complete
> 2023-04-05 20:17:20,984 INFO exited: valheim-server (exit status 0; expected)

And then it starts again.

**Comment by dylanReinsma** [comment]
I am having a similar issue after adding plus to my world... same seg fault error

**Comment by RobertLHarris** [comment]
Can you share your configs/env files?  I just got my server up and can't get VH/BepInEx working at all.  I just found the environment variable to set for VALHEIM_PLUS=TRUE, but when I enable it I get a client incompatibility.
